docs/local-dependencies.md

```diff
@@ -0,0 +1,38 @@
+# Local dependencies
+
+## Python and virtual environment
+
+It is recommended to work within a virtualenv to ensure proper dependencies isolation.
+If you're not familiar with that concept, read [Python Virtual Environments - a Primer][].
+
+Alright, now you can [install virtualenv][install-virtualenv] and then type these commands knowing what you are doing:
+
+```shell
+$ virtualenv --python=python2.7 venv
+$ source venv/bin/activate
+$ pip install -r udata/requirements/develop.pip
+$ pip install --no-deps -e udata/
+```
+
+
+## NodeJS and modules
+
+You need NodeJS 4.2. If it's not already installed or you have a different version,
+you should consider [installing NVM][nvm-install]. Then install JavaScript dependencies:
+
+```shell
```

Comment:
May be add for those having nvm: `nvm install` the first time and `nvm use` because there is a `.nvmrc` file